packages/coding-standards/tests/Unit/CsFixer/ForbiddenPrivateVisibilityFixer/config.yml

```diff
@@ -1,3 +1,6 @@
+imports:
+    - { resource: '../../../config.yml' }
```

Comment:
is it not possible to do this via https://github.com/shopsys/shopsys/blob/ph-travis-ecs-fix/packages/coding-standards/tests/autoload.php ?
so no repetition will be needed ?
will that be a problem when testing via monorepo ?

Reply:
The autoload.php is on a lower level then Symfony DIC build, so I believe it!s not possible to resolve that there.

In monorepo, everything is fine (you can see on our CI that tests pass).